eng/build-test.yml

```diff
@@ -0,0 +1,183 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osIdentifier: ''
+  container: ''
+  testGroup: ''
+  readyToRun: false
+  crossrootfsDir: ''
+  # If true, run the corefx tests instead of the coreclr ones
+  corefxTests: false
+  displayNameArgs: ''
+  condition: true
+
+### Test build job
+
+### Each test build job depends on a corresponding build job with the same
+### buildConfig and archType.
+
+jobs:
+- template: xplat-job.yml
+  parameters:
+    buildConfig: ${{ parameters.buildConfig }}
+    archType: ${{ parameters.archType }}
+    osGroup: ${{ parameters.osGroup }}
+    osIdentifier: ${{ parameters.osIdentifier }}
+    helixType: 'build/tests/'
+
+    # Test jobs should continue on error for internal builds
+    ${{ if eq(variables['System.TeamProject'], 'internal') }}:
```

Comment:
I have a feeling we do not want to continue on error for test build issues. @AaronRobinsonMSFT opinions?